### PR TITLE
More tiles and some cleanup

### PR DESCRIPTION
**Pull request by AJCM-git**
## About the PR 
-  Add useful tiles for mapping (bar, laundry, mime, clown, chapel, kitchen, snow and grass)
- Added tile item entities for every tile and carpet with their respective sprite and inhands
- Reorganized tiles prototypes and tile item entities prototypes
- Fixed weird joke with the pink carpet
- Fixed case of tiles display names
- Added some TODOs related to #5567 and #5188

**Screenshots**
image

### CODE REVIEW

**Comment by mirrorcult** [comment]
GRASS

**Comment by DogZeroX** [comment]
GRASS

**Comment by AJCM-git** [comment]
Marked as a draft by accident, oops (also fixed the linter, hopefully)

**Comment by moonheart08** [comment]
GRASS

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
Can't see licences / copyright on new tiles?

**Comment by AJCM-git** [comment]
@metalgearsloth new tile items are either handmade or taken from the same source the meta.json already points to. Didn't add copyright to the tiles before because the only one that has it is the steel tile (and they need to use RSIs anyways) so now i only added it to the ones i added/made

File: Resources/Textures/Tiles/copyright.txt [hunk 1]
```diff
@@ -1,3 +1,7 @@
 Alright because tiles don't use RSIs (yet) I gotta write this out.
 
-floor_steel.png taken from CEV-Eris commit 28e589f0ff72a009adf17db767e90be39054f0f2
+steel.png taken from CEV-Eris commit 28e589f0ff72a009adf17db767e90be39054f0f2
+
+bar.png, laundry.png, mime.png, clown.png and kitchen.png are under CC-BY-SA-3.0
```
Comment (metalgearsloth) [inline Resources/Textures/Tiles/copyright.txt:5]: Which codebase are these from? It's grouped with the tg ones but doesn't have a commit.

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**